modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -32,7 +33,12 @@ import {
 import {BAR_CHART, COLORS, DEFAULT_COLOR} from '../utils/constants';
 import {shortenNumber} from '../utils/shortenNumber';
 
-export default function AuditBarChart({rtl, vocabularies}) {
+const ESCAPE_KEYS = [
+	'Escape', // Most browsers.
+	'Esc', // IE and Edge.
```

Comment:
What versions of IE and edge are you supporting? If you need to support IE and Edge(non chromium), then you can't use `URLSearchParams `. So my guess is that either this support for `Esc` is unnecessary or we need to remove `URLSearchParams`.

Reply:
As I said, we are not going with IE but seems that we support Edge (latest), right? See https://caniuse.com/?search=URLSearchParams. With Edge (latest) we can use `URLSearchParams`, right? Or maybe I'm wrong 🤔

Reply:
You could use it with IE too if you wanted, because we polyfill it on older branches via [core-js](https://github.com/zloirock/core-js/issues/117#issuecomment-446230418), which we set up [here](https://github.com/liferay/liferay-portal-ee/blob/7.3.x/modules/apps/frontend-compatibility/frontend-compatibility-ie/build.gradle#L31-L32). ie. nothing to worry about.

Reply:
I missed ask that, because I see more `URLSearchParams` in many places in `liferay-portal` code. Thanks! ❤️